fdbserver/Ratekeeper.actor.cpp

```diff
@@ -732,7 +874,8 @@ void Ratekeeper::updateRate(RatekeeperLimits* limits) {
 		}
 
 		if (minSSVer != std::numeric_limits<Version>::max() && maxTLVer != std::numeric_limits<Version>::min()) {
-			// writeToReadLatencyLimit: 0 = infinite speed; 1 = TL durable speed ; 2 = half TL durable speed
+			// writeToReadLatencyLimit: 0 = infinite speed; 1 = TL durable speed ; 2 = half TL durable
```

Comment:
Strange that the comment formatting changed but the CI isn't complaining about formatting

Reply:
yeah, interesting